test

```diff
@@ -0,0 +1 @@
+line1edit
```

Comment:
Can you put the actual friken new line here please. Dunno why we need to. Some C problem apparently